src/screens/swap/swap-form/swap-percentage/swap-percentage.tsx

```diff
@@ -0,0 +1,110 @@
+import { TouchableOpacity } from '@gorhom/bottom-sheet';
+import { BigNumber } from 'bignumber.js';
+import { FormikState } from 'formik';
+import React, { FC, useState, useEffect, useMemo } from 'react';
+import { Keyboard, KeyboardAvoidingView, Text, View } from 'react-native';
+
+import { isAndroid } from '../../../../config/system';
+import { SwapFormValues } from '../../../../interfaces/swap-asset.interface';
+import { useTokensListSelector } from '../../../../store/wallet/wallet-selectors';
+import { formatSize } from '../../../../styles/format-size';
+import { getTokenSlug } from '../../../../token/utils/token.utils';
+import { useSwapPercentageStyles } from './swap-percentage.styles';
+
+interface Props {
+  formik: FormikState<SwapFormValues> & {
+    setFieldValue: (field: string, value: unknown, shouldValidate?: boolean) => void;
+  };
+}
+
+export const SwapPercentage: FC<Props> = ({ formik }) => {
```

Comment:
this also should be available on Send screen.  
Current implementation is not reusable.